### PR TITLE
Further tweaks from the reviews

### PR DESCRIPTION
**Pull request by Wiuwiti**

### CODE REVIEW

File: Evaluation.css
Comment (softvision-alexcarcea) [inline Evaluation.css]: This can be bundled up with the `margin` rule on L66.

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: I assume the left / right margins are user agent defaults? In that case, might want to consider bundling these up into a single `margin` rule, if it makes sense to you as well.

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: How did you come up with these particular percentage values (L171, L175, L180)?

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: I believe Sebi had a comment on another PR, stating that it helps better understand the structure and behavior of your stylesheets if you order your selectors in their natural order of appearance in the DOM. So, this particular ruleset might be better placed just below L160. Optional, since this will still work, but a good practice which improves legibility, nonetheless.

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css:201]: Whoa! Quite a bit of margin!

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: As opposed to `.TopFooter`? :stuck_out_tongue:

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: Why the underlines here? (I'm going to keep asking until I get an answer! :stuck_out_tongue:)

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: I, for one, am not terribly fond of CSS expressions, for multiple reasons:
* they violate separation of concerns
* they pollute the CSS with overly verbose computations
* they're non-standard CSS
* support has been [discontinued since IE8](https://msdn.microsoft.com/en-us/library/ms537634(v=vs.85).aspx)

File: NewEvaluation.css
Comment (softvision-alexcarcea) [inline NewEvaluation.css]: Might I suggest breaking these rules on multiple lines for improved readability?

File: NewEvaluation.html
Comment (softvision-alexcarcea) [inline NewEvaluation.html:646]: What do you suppose is going to happen when this submit button is clicked?

**Review by softvision-alexcarcea — CHANGES_REQUESTED** [review]
I like the cleanup, but there's just a bit more cleanup (around `_`-prefixed CSS properties and usage of CSS expressions) that I feel is needed before this PR gets merged in.

File: NewEvaluation.css
Comment (Wiuwiti) [reply NewEvaluation.css]: I noticed initially that the tables is divided in 3 columns, there is where  I got 33% from. This 33% would be for a pair of label and dropdown menu so now we have the parent that is 1/3 of the width of the parent and has 2 elements that should be half each. Initial when I made the code I didn't check the initial values of paddings margins and borders of the elements and as a go-around method I gave them some aproximation values of 40% and 49% that would align them in the column the way I wanted

Comment (Wiuwiti) [reply NewEvaluation.css]: Foound out why I couldn't do the 33% then 50% 50%, the drop down box menu has a initial padding of 3px around. Wanted also to know if there is a way to make the width with % values or VW but also add and subtract pixels ? something like 50% -3px in this case

Comment (softvision-alexcarcea) [reply NewEvaluation.css]: There's `calc()` and there's `box-sizing`. In your case, you probably want to look at the latter before anything else, though the former may come in handy every now and then, too.

File: NewEvaluation.css
Comment (Wiuwiti) [reply NewEvaluation.css]: I used this piece of code more or less because I had issues with placing the footer the way I desired. (sorry for late responses )

Comment (softvision-alexcarcea) [reply NewEvaluation.css]: And did it actually work the way you intended it to work? I'm really curious, because a) I've never played around with CSS expressions myself and b) I still find the extra `_` before the property name rather dubious.

Comment (Wiuwiti) [reply NewEvaluation.css]: Nope in the end I started playing with this without the expression. Managed to make it work the same  way it worked with the expression. Don't know why it acted and worked differently before

File: NewEvaluation.html
Comment (Wiuwiti) [reply NewEvaluation.html:646]: All the data that has been entered in the forms to be sent to the server ? and then to be sent to the Evaluation page where u can see the entred data

Comment (softvision-alexcarcea) [reply NewEvaluation.html:646]: Yes. Well, sort of... You see, in HTML, when you submit a form, you only submit the data that is _part of that particular form_. Multiple forms? Fragmented data. So, then, what's happening in this specific case? The only data that gets sent to the "backend" (currently, `Evaluation.html`) is a parameter named `button` with a value of `Submit`. But what about the rest of the data that you've input into other forms? Well, tough luck... it's now gone. :)

File: Evaluation.html
Comment (softvision-alexcarcea) [inline Evaluation.html:8]: What does the `T` stand for?

**Review by softvision-alexcarcea — APPROVED** [review]

File: Evaluation.html
Comment (Wiuwiti) [reply Evaluation.html:8]: Theme , the menthor that came today proposed us to divide our css code into thematical css classes that work with color text size and overall feel of the button like a skin  , and localion wise that modifies the position and height stuff about the place and size

File: NewEvaluation.html
Comment (Wiuwiti) [reply NewEvaluation.html:646]: So i should find a way to hook all the data and aend it in a specific way to the server ?

File: Evaluation.html
Comment (softvision-alexcarcea) [reply Evaluation.html:8]: OK, got it, thanks for the clarification. :wink:

File: NewEvaluation.html
Comment (softvision-alexcarcea) [reply NewEvaluation.html:646]: Yup, that's more or less what I'm suggesting. Not the data processing part just yet, naturally (since we've just started the JS basics), but just the markup boilerplate that will allow us to pass the appropriate data to the backend.

File: Evaluation.html
Comment (Wiuwiti) [reply Evaluation.html:8]: Just want to know also if is ok how I put the idea in practice. I don't know if I did already because this is the first time I have tried this and it seems a little bit odd for me

Comment (softvision-alexcarcea) [reply Evaluation.html:8]: I'll admit that I'm more familiar with the type of separation that Gabi is suggesting from my time as a Flex developer than after moving to the JS / HTML / CSS front, but from what I can tell, what you did thus far looks good. I'd defer to Gabi for a final answer though, since I've yet to sync up with him regarding his expectations on this.

**Pull request merged**
